### ISSUE TITLE
xcode7.1: iOS testing builds hanging on 'Touching xyz.tests"

### ISSUE THREAD

**Issue by scottrhoyt** [travis-ci/travis-ci#5020]
Since updating to the xcode7.1 image, I have noticed an increase in the number of builds that are hanging on the point where they touch the test target. This eventually will cause the build to fail because it times out.  This happens on about 10%-20% of the builds.

I am incorporating the workarounds as discussed in #4906.

Any help would be appreciated, thanks!

**Comment by cotsog** [comment]
Hey @scottrhoyt!

Did you implement [this](https://github.com/travis-ci/travis-ci/issues/4906#issuecomment-146985743) in order to have visibility on error messages that can be hidden by `xcpretty`?

Can you also send us a link to a failing build for us to investigate at support (at) travis-ci.com?

Thanks!

**Comment by scottrhoyt** [comment]
No, I haven't.  The issue is hard to reproduce. It happens intermittently on builds about 10-20% of the time. When the same build is restarted, it often succeeds. I will wait until I get another failing build to send you an email.
